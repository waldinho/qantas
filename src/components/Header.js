import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`
    .logo {
        background: url('../img/qantas-logo.png');
        width:100px;
        height: 100px;
    }
`;

const Header = () => {

    return(
        <Wrapper>
            <header className='App-header'>
                <img src="https://www.qantas.com/content/dam/qantas/centenary/QANTAS-Centenary-40px.png" alt='Qantas' title='Qantas'/>
            </header>
        </Wrapper>  
    );
}

export default Header