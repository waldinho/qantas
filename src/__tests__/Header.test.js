import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Header from '../components/Header';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
  ReactDOM.unmountComponentAtNode(div);
});

describe('<Header /> with no props', () => {
  const container = shallow(<Header />);
  it('should match the snapshot', () => {
    expect(container.html()).toMatchSnapshot();
  })
})
