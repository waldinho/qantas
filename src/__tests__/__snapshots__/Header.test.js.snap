// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> with no props should match the snapshot 1`] = `"<div class=\\"sc-bdVaJa cBvsrr\\"><header class=\\"App-header\\"><img src=\\"https://www.qantas.com/content/dam/qantas/centenary/QANTAS-Centenary-40px.png\\" alt=\\"Qantas\\" title=\\"Qantas\\"/></header></div>"`;
